package.json

```diff
@@ -19,7 +19,7 @@
     "debug": "2.6.9",
     "depd": "2.0.0",
     "destroy": "1.2.0",
-    "encodeurl": "~1.0.2",
+    "encodeurl": "~2.0.0",
```

Comment:
@UlisesGascon Please do not forget this before doing a new express security release.

The `send` with this merge request should be published first on npm. Thank you.